app/views/contacts/create.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#new_contact').css("display","none").fadeOut(1000);
```

Comment:
no need to use `display, none` if you use `#fadeOut`

> The .fadeOut() method animates the opacity of the matched elements. Once the opacity reaches 0, the display style property is set to none, so the element no longer affects the layout of the page.

Reply:
fixed, thanks